low-level/files/opt/seagate/sspl/conf/sspl.init.conf.tmpl

```diff
@@ -0,0 +1,47 @@
+cluster:
+  cid01:
+    name: cortx_cluster
+    site:
+      storage_set_count: '1'
+    site_count: '1'
+    storage_set:
+    - {}
+    - name: StorageSet-1
```

Comment:
>cluster>{cluster-id}>storage_set[N]>name

storage set would be an array and 'name', 'storage_enlcosures' per array element

Reply:
Addressed this.